src/annotator/test/guest-test.coffee

```diff
@@ -238,7 +238,7 @@ describe 'Guest', ->
       guest = null
 
       beforeEach ->
-        sandbox.stub(document, 'title', 'hi')
+        document.title = 'hi'
```

Comment:
Can't say I'm wonderfully happy about this, but sinon logs a warning that the previous `sandbox.stub(document, 'title', 'hi')` is deprecated, and `sandbox.stub(document, 'title').value('hi')` (which I'd think would be correct) doesn't work (the code gets `undefined` instead of `"hi"` for `document.title` and fails the test).